mono/mini/driver.c

```diff
@@ -142,6 +142,7 @@ extern char *nacl_mono_path;
 	MONO_OPT_GSHARED |	\
 	MONO_OPT_SIMD |	\
 	MONO_OPT_ALIAS_ANALYSIS	| \
+	MONO_OPT_DIVREM |	\
```

Comment:
Add it to some of the test sets and, after a while in master, add it to the default set.